components/playlist/browser/playlist_service.cc

```diff
@@ -265,8 +265,9 @@ bool PlaylistService::MoveItem(const PlaylistId& from,
     return false;
   }
 
-  NotifyPlaylistChanged({PlaylistChangeParams::Type::kItemDeleted, *from});
-  NotifyPlaylistChanged({PlaylistChangeParams::Type::kItemAdded, *to});
+  NotifyPlaylistChanged({mojom::PlaylistEvent::kItemDeleted, *from});
+  NotifyPlaylistChanged({mojom::PlaylistEvent::kItemAdded, *to});
```

Comment:
Q. I thought you would remove these here when we talked about making kItemMoved. Was there any problem with that?